.github/workflows/containers.yml

```diff
@@ -17,23 +17,30 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container: [ functional, integration, unit ]
+        distro: [ centos-8, centos-9, alma-9 ]
+        include:
+          - directory: docker/network
+            prefix: ovirt/vdsm-network-tests
+            container: [ functional, integration, unit ]
```

Comment:
This is not ideal either, if I'm not mistaken it will create 9 jobs for network, building the same images three times (e.g., centos-8-function, centos-8-integration, and centos-8-unit will be three jobs all of them building the same images), and then pushing them individually.

Maybe we should keep both containers (test and network) in different jobs after all.